Mathlib/NumberTheory/NumberField/ClassNumber.lean

```diff
@@ -18,16 +19,27 @@ on the class number.
 ## Main definitions
 - `NumberField.classNumber`: the class number of a number field is the (finite)
 cardinality of the class group of its ring of integers
-- `RingOfIntegers.isPrincipalIdealRing_of_isPrincipal_of_mem_primesOver_of_le`: to show that the
-ring of integer of a number field is a PID it is enough to show that all ideals above any (natural)
-prime `p` smaller than Minkowski bound are principal. This is the standard technique to prove that
-`𝓞 K` is principal, see [marcus1977number], discussion after Theorem 37.
+- `isPrincipalIdealRing_of_isPrincipal_of_le_pow_inertiaDeg_of_mem_primesOver_of_mem_Icc`: let `K`
+be a number field and let `M K` be the Minkowski bound of `K`. To show that `𝓞 K` is a PID it is
```

Comment:
Since it is not defined explicitly anywhere in the library, can you say here what it is and point at its def in this file?